front50-core/src/main/groovy/com/netflix/spinnaker/front50/validator/HasEmailValidator.java

```diff
@@ -0,0 +1,17 @@
+package com.netflix.spinnaker.front50.validator;
+
+import com.google.common.base.Strings;
+import com.netflix.spinnaker.front50.model.application.Application;
+import java.util.Optional;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HasEmailValidator implements ApplicationValidator {
+  @Override
+  public void validate(Application application, ApplicationValidationErrors validationErrors) {
+    if (Strings.isNullOrEmpty(Optional.ofNullable(application.getEmail()).orElse("").trim())) {
```

Comment:
I think you could either `Strings.isNullOrEmpty(application.getEmail())` or `Optional.ofNullable(application.getEmail()).orElse("").trim().isEmpty()`